lib/ui/platform.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart_ui;
+
+/// Platform services provided to the application by Flutter.
+///
+/// Invoke a platform service by passing a request as json.
+
+typedef void PlatformServiceReply(String json);
+
+void platformService(String json, PlatformServiceReply callback)
```

Comment:
We usually use verbs for functions.  `sentToPlatformService`

Also, the previous iteration had a `messageName` string in addition to the payload.  That was useful for routing the message later in the pipeline (without needing to unpack the payload).